packages/amplify-cli/src/commands/env/remove.js

```diff
@@ -1,16 +1,21 @@
 const fs = require('fs-extra');
+const path = require('path');
+const ora = require('ora');
 
 module.exports = {
   name: 'remove',
   run: async (context) => {
-    const envName = context.parameters.options.name;
+    const envName = context.parameters.first;
+    const currentEnv = context.amplify.getEnvInfo().envName;
+
     if (!envName) {
       context.print.error('You must pass in the name of the environment using the --name flag');
```

Comment:
need to change this line because it's inconsistent with the help message.
if help message says 
`remove <env-name>`
It means there's no --name flag anymore, and the first parameter <env-name> is mandatory.

Reply:
good catch. Will change.